src/nemos/glm.py

```diff
@@ -1116,6 +1117,9 @@ def _get_optimal_solver_params_config(self):
         """Return the functions for computing default step and batch size for the solver."""
         return glm_compute_optimal_stepsize_configs(self)
 
+    def __repr__(self):
+        return format_repr(self, ["feature_mask"])
```

Comment:
```suggestion
        return format_repr(self)
```
Since format_repr now doesn't show arrays